addons/spreadsheet/static/tests/links/dashboard_links_plugin_tests.js

```diff
@@ -24,7 +25,7 @@ function beforeEach() {
     registry.category("services").add("orm", ormService, { force: true }); // #action-serv-leg-compat-js-class
 }
 
-QUnit.module("spreadsheet_dashboard > link", { beforeEach });
+QUnit.module("spreadsheet > link", { beforeEach });
```

Comment:
Why ? We are still in spreadsheet_dashboard module

Reply:
no.
The file name should be changed